src/troute-network/troute/HYFeaturesNetwork.py

```diff
@@ -886,3 +885,78 @@ def replace_waterbodies_connections(connections, waterbodies):
             new_conn[n] = connections[n]
     
     return new_conn, link_lake
+
+def get_rfc_lake_gage_crosswalk():
+    """
+    Temporary solution for loading rfc gage-lake crosswalk information. This prevents having to 
+    read this info from a file (e.g., a .csv file). This will not be needed once this information
+    is added to the hydrofabric.
+    """
+    rfc_dict = {'rfc_gage_id': {347987: 'KNFC1', 361273: 'IRGC1', 371555: 'HEYO2', 389804: 'CNLO2', 
```

Comment:
If this will eventually be an external resource, could we make the temporary fix also an external file and import(e.g., `from rfc_lake_gage_crosswalk import rfc_dict`), rather than baking the dictionary into this file? It would be easy for this to linger for a while.

Reply:
These changes have been made.